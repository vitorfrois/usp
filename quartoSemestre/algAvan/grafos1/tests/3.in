10 20 10
2 4 5
2 7 8
1 2 3
9 10 6
8 9 6
8 10 9
1 6 6
2 5 7
6 9 7
3 4 9
4 8 2
5 6 2
7 8 5
5 9 8
7 9 7
4 6 8
2 3 6
6 7 10
4 9 3
4 5 4
3 2
9 8
10 4
5 9
6 10
8 9
4 10
1 2
10 2
6 10
