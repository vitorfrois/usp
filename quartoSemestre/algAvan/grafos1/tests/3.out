6
5
9
7
13
5
9
3
14
13
